src/web_ui/src/components/Pages/__tests__/Homepage.test.js

```diff
@@ -1,32 +1,23 @@
 import React from 'react';
-import { render, unmountComponentAtNode } from 'react-dom';
-import { act } from "react-dom/test-utils"
+import { configure, shallow } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+import Album from "../Homepage";
 
-import HomePage from "../Homepage"
+configure({ adapter: new Adapter() });
```

Comment:
use the `render` and `screen` methods from react testing library to run the tests